app.py

```diff
@@ -218,8 +218,9 @@ def get_state(code_):
 @app.route('/states', methods=['GET'])
 def get_states():
     states = State.query.all()
+    sorted_states = sorted(states, key=lambda state: state.total, reverse=True)
```

Comment:
By default, states will be returned in descending order based on their total, so California, with a total of 29, will be first, all the way down to Virginia with a total of 0 :(